CHANGELOG.md

```diff
@@ -155,6 +144,21 @@ function ProtectedHomeComponent() {
 }
 ```
 
+## [0.24.4] - 2022-08-07
+
+-   Fixes styling issue for login with github
+
+## [0.24.3] - 2022-08-05
+
+### Changed
+
+-   Updated `react-scripts` in all examples
+-   Updated READMEs for example apps, removed information that is no longer relevant.
+
+### Fixed
+
+-   Fixed capitalization of GitHub on the UI (fixes https://github.com/supertokens/supertokens-auth-react/issues/539)
+
```

Comment:
Not sure why this is showing. Is the base branch properly merged with the latest?

Reply:
This fixes a merge error from the base branch. I can commit this "fix" there directly if you prefer.